js/options.js

```diff
@@ -103,4 +133,23 @@ function getBlacklistedDomainsFromSource()
         }
     }
     return {"timestamp":0,"domains":[]};
+}
+
+function getWhitelistedDomainsFromSource()
+{
+    console.log("Getting whitelist from GitHub now");
+    var objAjax = new XMLHttpRequest();
+    objAjax.open("GET", "https://raw.githubusercontent.com/409H/EtherAddressLookup/master/whitelists/domains.json", true);

```

Comment:
Already pushed that file to `master` :) https://github.com/409H/EtherAddressLookup/commit/bbf7f54f1cdd1d094305c1f47f1c5cc166b79ed5